test/parallel/test-print-GH-23191.js

```diff
@@ -0,0 +1,43 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
We don't add the copyright header in new tests.

Reply:
Thanks. Fixed it.